content/blog/ways-to-interact-with-kubernetes-clusters-managed-by-hpe-ezmeral-container-platform.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Ways to interact with Kubernetes Clusters managed by HPE Ezmeral
+  Container Platform
+date: 2021-05-28T08:40:17.101Z
+author: Cenz Wong
+authorimage: /img/Avatar5.svg
+thumbnailimage: https://user-images.githubusercontent.com/72959956/119960940-128cf180-bfd8-11eb-9018-8411eda59a91.png
+---
+
+HPE Ezmeral Software Platform has a bundle of software that helps you run, manage, control, and secure the apps, data, and IT that run your business. One of which is the HPE Ezmeral Container Platform (HPE ECP). It is a unified cloud container software platform built on Kubernetes. How exactly you can connect to HPE ECP to interact with K8s? Don't panic! This blog post will introduce to you most, if not all, of the ways to connect with the Kubernetes inside HPE ECP.

```

Comment:
This introductory paragraph should be deleted. It is a duplicate paragraph of the next paragraph that is correct and took into account Dale's suggestion.

Reply:
I concur with Denis.